etcd/watchmanager/types.go

```diff
@@ -35,7 +35,7 @@ type WatchManager interface {
 }
 
 // UpdateFn is called when an event on the watch channel happens
-type UpdateFn func(key string) error
+type UpdateFn func(key string, events []*clientv3.Event) error
```

Comment:
It makes sense to have all UpdateFns to use the events instead of re-geting. I rather leave this and have the option to use it later.